README.md

```diff
@@ -1,2 +1,4 @@
-# triton-touchbase
-Touchbase stack designed for container-native deployment on Joyent's Triton platform
+triton-elasticsearch
```

Comment:
Oops, copypasta problem.

We're not distributing the Touchbase code as our own in this case, merely installing it as part of our application. At most we might include the license for Touchbase within the LICENSE file.